spec/python-spec.coffee

```diff
@@ -278,3 +278,28 @@ describe "Python grammar", ->
     expect(tokens[4][0]).toEqual value: ')', scopes: ['source.python', 'meta.function.python', 'punctuation.definition.parameters.end.python']
     expect(tokens[4][1]).toEqual value: ':', scopes: ['source.python', 'meta.function.python', 'punctuation.section.function.begin.python']
 
+
+  it "correctly enables SQL inline highlighting on blocks", ->
+    tokens = grammar.tokenizeLines('''
```

Comment:
Fun fact!  You can actually combine these two specs by using a for loop.  See https://github.com/atom/language-javascript/blob/master/spec/javascript-spec.coffee#L20-L32 as an example.